src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Ball {
+

```

Comment:
[Please don't add redundant empty lines to your code.](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code)